lua/ttt2/libraries/tips.lua

```diff
@@ -72,7 +72,7 @@ function tips.Initialize()
     tips.Register("tip31")
     tips.Register("tip32")
     tips.Register("tip33")
-    tips.Register("tip34", { helpkey = Key("+gm_showhelp", "F1") })
```

Comment:
Why is this change in here?

Reply:
Because I noticed that this is unused and I didn't want to open a PR for this single line

Reply:
Okay but this is in general confusing as it is also not mentioned in the pr description.